src/codewhisperer/util/supplementalContext/crossFileContextUtil.ts

```diff
@@ -158,21 +166,31 @@ async function getRelevantCrossFiles(editor: vscode.TextEditor, dependencyGraph:
         return isRelevant(editor.document.fileName, file, editor.document.languageId)
     })
 
-    const relevantOpenFiles: vscode.Uri[] = await getRelevantFilesFromEditor(
-        editor.document.fileName,
-        editor.document.languageId
-    )
+    const mergedCrossFileList = [...new Set([...srcDependencies, ...samePackageRelevantFiles])]
 
-    // We refer to only those open files which are in srcDependencies
-    const filteredRelevantOpenFiles = relevantOpenFiles
-        .filter(file => srcDependencies.includes(file.fsPath))
-        .map(file => file.fsPath)
+    return mergedCrossFileList
+}
 
-    const mergedCrossFileList = [
-        ...new Set([...filteredRelevantOpenFiles, ...srcDependencies, ...samePackageRelevantFiles]),
-    ]
+// Util to move selected files to the front of the input array if it exists
+function moveToFront<T>(arr: T[], picked: Set<T>) {
+    return [...arr].sort((a, b) => (picked.has(b) ? 1 : 0) - (picked.has(a) ? 1 : 0))
+}
 
-    return mergedCrossFileList
+function openFilesInWindow(): string[] {
```

Comment:
Could you rename this to `getCurrentOpenFilesInWindow` or similar and write a doc string to explain what the return object is?

`openFilesInWindow` seems like a name that opens a file as a new tab. But what you are doing is to get the file path of all opened tabs.

Reply:
sg